ports/mimxrt/machine_rtc.c

```diff
@@ -25,15 +25,126 @@
  * THE SOFTWARE.
  */
 
+#include "py/mperrno.h"
 #include "py/runtime.h"
+#include "shared/runtime/mpirq.h"
 #include "shared/timeutils/timeutils.h"
 #include "modmachine.h"
 #include "ticks.h"
 #include "fsl_snvs_lp.h"
 
+STATIC mp_int_t timeout = 0;
+
+void machine_rtc_alarm_clear_en() {
```

Comment:
please put `void` inside empty parenthesis for empty argument lists (there are more cases of this below)